homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -66,14 +69,17 @@
                         Envoy(
                             ip_address,
                             condition,
-                            "{} {}".format(SENSORS[condition][0], inverter),
+                            "{}{} {}".format(name, SENSORS[condition][0], inverter),
                             SENSORS[condition][1],
                         )
                     )
         else:
             entities.append(
                 Envoy(
-                    ip_address, condition, SENSORS[condition][0], SENSORS[condition][1]
+                    ip_address,
+                    condition,
+                    name + SENSORS[condition][0],
```

Comment:
Please use f-strings instead of string concatenation.

Reply:
Done